rqt_gui_py/src/rqt_gui_py/ros_py_plugin_provider.py

```diff
@@ -65,8 +68,11 @@ def _init_node(self):
         if not self._node_initialized:
             name = 'rqt_gui_py_node_%d' % os.getpid()
             qDebug('RosPyPluginProvider._init_node() initialize ROS node "%s"' % name)
+
             rclpy.init()
             self._node = rclpy.create_node(name)
```

Comment:
A possible improvement: Add a ReentrantCallbackGroup to the context so that multiple plugins can have callbacks serviced simultaneously.

Reply:
A `ReentrantCallbackGroup` implies that the very same callback can be called multiple times (not that "multiple plugins can have callbacks serviced simultaneously"). Usually that is undesired and most callback function won't be written in a way to support that.

Reply:
That's good to know Dirk. We'll close the issue.